src/compiler/checker.ts

```diff
@@ -9689,6 +9692,7 @@ namespace ts {
                     type = errorType;
                 }
                 symbol.type = type;
+                symbol.mapper = undefined!;

```

Comment:
I admit to being curious about `undefined!`.

Reply:
We do that in a few other places to drop object references such that they memory can be collected. Basically, the `mapper` exists only to store a mapper used for lazy-computing the `type` property. It is never accessed thereafter. We could of course change the `mapper` type annotation to include `undefined`, but then we'd just have to spread `!`s elsewhere.